1080i/script-skin_helper_service-ColorPicker.xml

```diff
@@ -30,28 +30,28 @@
                 <scrolltime>200</scrolltime>
                 <orientation>vertical</orientation>
                 <preloaditems>2</preloaditems>
-                <itemlayout width="150" height="150">
+                <itemlayout width="128" height="128">
                     <control type="image">
                         <left>0</left>
                         <top>0</top>
-                        <right>10</right>
-                        <bottom>10</bottom>
+                        <right>8</right>
+                        <bottom>8</bottom>
                         <texture colordiffuse="dialog_fg_12" border="5">common/colorpick.png</texture>
                     </control>
                     <control type="image">
-                        <top>10</top>
-                        <left>10</left>
-                        <right>20</right>
-                        <bottom>20</bottom>
+                        <top>8</top>
+                        <left>8</left>
+                        <right>16</right>
+                        <bottom>16</bottom>
```

Comment:
Same here. Can you revert the padding top/left pad to 10 and right/bottom pad to 20